sql-statements/sql-statement-show-placement-for.md

```diff
@@ -11,7 +11,13 @@ summary: The usage of SHOW PLACEMENT FOR in TiDB.
 >
 > If you understand the risks, you can enable this experiment feature by executing `SET GLOBAL tidb_enable_alter_placement = 1;`.
 
-`SHOW PLACEMENT FOR` summarizes all placement options from direct placement and placement policies, and presents them in the canonical form for a specific table, database schema, or partition.
+`SHOW PLACEMENT FOR` summarizes all placement options, and presents them in the canonical form for a specific table, database schema, or partition.
```

Comment:
placement options, policies, or rules?

Reply:
- A placement OPTION is something like "FOLLOWERS=4".
- A placement POLICY is a way of grouping options together into a named object which can be assigned to tables.
- Placement RULES previously included both direct placement options and placement policies, but we've since removed direct placement so this term has lost much of its meaning. So it is now usually possible to use RULES and POLICY interchangably, but I think we should keep the terms for now because it is similar to using the terms "RBAC" and "permissions" interchangebly. "Policy" is like the RBAC of Placement.